File: app/src/main/java/hungpt/development/DaNhan.java
package hungpt.development;

/**
 * Created by Hung Phan on 4/18/2017.
 */

public class DaNhan {
    String SoTK;
    String SoTKNhan;
    String MaHH;
    String Username;
    String MoTa;
    String VTN;
    String VTD;
    String ThoiGian;
    String TTDon;

    public DaNhan(String soTK, String soTKNhan, String maHH, String username, String moTa, String VTN, String VTD, String thoiGian, String TTDon) {
        SoTK = soTK;
        SoTKNhan = soTKNhan;
        MaHH = maHH;
        Username = username;
        MoTa = moTa;
        this.VTN = VTN;
        this.VTD = VTD;
        ThoiGian = thoiGian;
        this.TTDon = TTDon;
    }
}

File: MyShipper_server/DaNhan.php
<?php
 include_once("connection.php");

   $statement = mysqli_prepare($conn, "SELECT donhang.SoTk, nhanship.SoTKNhan , nhanship.MaHH,
     Username, MoTa, VTN, VTD, ThoiGian ,TTDon , nhanship.TGNhan, LuotXemShip FROM `khachhang`,
     `donhang` , `nhanship` WHERE donhang.MaHH = nhanship.MaHH and donhang.SoTK = khachhang.SoTK ORDER BY TGNhan DESC");

   mysqli_stmt_execute($statement);
   mysqli_stmt_store_result($statement);
   mysqli_stmt_bind_result($statement, $SoTK,$SoTKNhan, $MaHH, $Username, $MoTa, $VTN, $VTD, $ThoiGian,$TTDon,$TGNhan, $LuotXemShip);
   $response["danhan"] = array();
   $tmp = array();

    while(mysqli_stmt_fetch($statement)){

    $tmp["SoTK"] = $SoTK;
    $tmp["SoTKNhan"] = $SoTKNhan;
    $tmp["MaHH"] = $MaHH;
	  $tmp["Username"] =$Username;
    $tmp["MoTa"] = $MoTa;
	  $tmp["VTN"] = $VTN;
    $tmp["VTD"] = $VTD;
    $tmp["ThoiGian"] = $ThoiGian;
    $tmp["TTDon"] = $TTDon;
    $tmp["TGNhan"] = $TGNhan;
    $tmp["LuotXemShip"] = $LuotXemShip;
    array_push($response["danhan"], $tmp);
    }
   echo json_encode($response);
?>
